Cargo.toml

```diff
@@ -9,12 +9,17 @@ repository = "https://github.com/jeremyletang/rust-sfml"
 license = "zlib-acknowledgement"
 
 [features]
+default = ["graphics", "audio", "network"]
```

Comment:
Note that `window` is implied by `graphics`, so it doesn't have to be explicitly included here.